### PR TITLE
task/LoginStrategy login to ssi-hub with siwe

### PR DESCRIPTION
**Pull request by JGiter**

### CODE REVIEW

File: lib/cacheServerClient.ts [hunk 1]
```diff
@@ -72,39 +72,68 @@ export class CacheServerClient {
         `DID Login Strategy is now logged into cache server after ${duration.toFixed()}ms`
       );
     });
-    const token = await retry.execute(async () => {
```
Comment (jrhender) [inline lib/cacheServerClient.ts:75]: We should keep this `retry`

File: lib/cacheServerClient.ts [hunk 2]
```diff
@@ -73,38 +73,36 @@ export class CacheServerClient {
...
+
+      const signature = await this.signer.signMessage(message);
+
+      const { data } = await this.httpClient.post<{ token: string }>(
+        '/login/siwe/verify',
+        {
+          message,
+          signature,
+        }
+      );
```
Comment (artursudnik) [inline lib/cacheServerClient.ts:98]: I'd add a debug log line in case of error. It will reduce pain for someone who will troubleshoot :)

Comment (JGiter) [reply lib/cacheServerClient.ts:98]: I think the error should be logged already in retry https://github.com/energywebfoundation/passport-did-auth/blob/d17bd1fc3b6bc581b34acb775e34f76830d9f858/lib/cacheServerClient.ts#L60

**Review by jrhender — APPROVED** [review]
Looks good to me 👍

**Pull request merged**